src/Main.js

```diff
@@ -12,7 +12,7 @@ return(
                     <Route path="/" exact>
                             <h2>Welcome to Chuck Norris's best jokes</h2>
                     </Route>
-                    <Route path="/random?category=:name">
```

Comment:
? defines a query parameter and not path. For a different page you have to use a different path

Reply:
thanks a lot Fabi :D